Add examples to clarify inventory plugin usage (#1347)
ansible/ansible-documentation: docs/docsite/rst/plugins/inventory.rst

To start using an inventory plugin with a YAML configuration source, create a file with the accepted file name schema documented for the plugin in question, then add ``plugin: plugin_name``. Use the fully qualified name if the plugin is in a collection.

> .. note:: Inventory plugins have required name patterns they must conform to. For example

Reviewer: ```suggestion
.. note:: Inventory plugins have required name patterns to which they must conform, for example:
```

>   An inventory using the ``kubevirt.core.kubevirt`` inventory plugin must have a filename ``*.kubevirt.yml``
>   An inventory using the ``servicenow.servicenow.now`` inventory plugin must have a filename ``*.servicenow.yml``

Reviewer: ```suggestion
  An inventory that includes the ``kubevirt.core.kubevirt`` inventory plugin must use the ``*.kubevirt.yml`` filename pattern. 
  An inventory that includes the ``servicenow.servicenow.now`` inventory plugin must use the ``*.servicenow.yml`` filename pattern.
```

> 

Reviewer: I also suggest removing some of the extra empty lines here.
